src/aat/resources/features/F-047/F-047_Test_Data_Base.td.json

```diff
@@ -0,0 +1,28 @@
+{
+  "_guid_": "F-047_Test_Data_Base",
+  "productName": "CCD Data Store",
+  "operationName": "Get case ids",
+  "method": "GET",
+  "uri": "/caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid}/cases/ids?userId={userid}",
```

Comment:
"uri": "/caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid}/cases/ids?userId={userid}"
Ideally, the parameter "userId={userid}" should be set using QueryParams.

Reply:
Updated.
    "queryParams": {
      "userid": "F047UserIdTest"
    }
  "uri": "/caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid}/cases/ids?userId"